cpp/src/aztec/honk/sumcheck/sumcheck_round.hpp

```diff
@@ -78,9 +78,21 @@ template <class FF, size_t num_multivariates, template <class> class... Relation
     {
         // FF's default constructor may not initialize to zero (e.g., barretenberg::fr), hence we can't rely on
         // aggregate initialization of the evaluations array.
-        std::fill(evaluations.begin(), evaluations.end(), 0);
+        std::fill(evaluations.begin(), evaluations.end(), FF(0));
     };
 
+    /**
+     * @brief After computing the round univariate, it is necessary to zero-out the accumulators used to compute it.
+     */
+    template <size_t idx = 0> void reset_accumulators()
```

Comment:
Was not resetting accumulators to zero after each round :facepalm:. Took a long time to catch this.